src/telemetry/metrics.yaml

```diff
@@ -1,44 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
----
-$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
```

Comment:
Please check if this is going to break the probe-scraper

Reply:
I have a bug for this already https://github.com/mozilla/probe-scraper/issues/716. I will resolve that bug first and then merge it.

Reply:
I'd recommend trying to dry-run the probe-scraper before merging this PR

Reply:
As we are leaving the metrics and pings in the tree, I believe we don't need to do this.